### PR TITLE
Update README with checklist for Iron Tutorial party

### PR DESCRIPTION
**Pull request by Yadunund**
You can see how the list is rendered here https://github.com/osrf/ros2_test_cases/tree/yadu/update_readme_for_iron

Clicking on any of the links should open the Issues tab with the appropriate label filters applied

### CODE REVIEW

**Comment by clalancette** [comment]
Overall comments on this README from the discussion:

* Rewrite to have it be a "getting started" guide.
* Add in language saying that if you have Windows, prefer to test that.
* Add in headings for "Core" vs "Features"
* Prioritize things, putting the most important "sets" at the top

**Comment by Yadunund** [comment]
Thanks for the feedback! Re-ordered the sets along with some general comments.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,500 @@
...
+**Encountered failures?**
+If one or more checks fail, please provide the error message in a codeblock comment or as a gist file attachment.
+
+## What to prioritize?
+The goal is to close all tickets after successful validation. But we would like to prioritize the following sets in order:
+
+> **Note**: If you have access to a `Windows` machine, please prioritize testing on `Windows`. The next priority is `RHEL-9` followed by `Ubuntu Jammy`.
+
+- [ ] [Tutorials from docs.ros.org](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label%3Adocs)
+  - [ ] [Installation on various plarforms](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label%3Ainstallation+)
```
Comment (clalancette) [inline README.md:35]:
```suggestion
  - [ ] [Installation on various platforms](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label%3Ainstallation+)
```

**Review by clalancette — CHANGES_REQUESTED** [review]
Other than my one comment inline, this looks pretty good to me.

@cottsay @mjcarroll @Voldivh can you please take a look as well?

**Review by clalancette — APPROVED** [review]

**Review by Voldivh — COMMENTED** [review]
LGTM! As someone that have never contributed to a Tutorial Party (or seen anything related to it), the initial instructions and structure of the priority gave me enough information in order to know what to do and where to start, so I believe the discussed goals were accomplished 👍  
 
As a side note, would it be worth to add a brief section like "Why contribute?" to motivate the community to participate? Or is it explicitly enough by just stating it is a Tutorial Party?

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
> As a side note, would it be worth to add a brief section like "Why contribute?" to motivate the community to participate? Or is it explicitly enough by just stating it is a Tutorial Party?

@Voldivh That's not a bad idea; @Yadunund , should we consider this?

Besides that, I think this looks pretty good, so I'm going to merge this so that the front page is not *too* out-of-date.  We can always do a follow-up PR here to beef up the front README.

**Pull request merged**